BackTranslation/measure_blue_score.py

```diff
@@ -13,7 +13,7 @@
 
 
 def measure_bleu_score(args, model, tokenizer, data_source):
-    translation_task = f"translation_{args.source_lang_id}_to_{args.target_lang_id}"
+    translation_task = f"translation_{args.data.source_lang_id}_to_{args.data.data.target_lang_id}"
```

Comment:
Why is the second one `args.data.data.target_lang_id` and not just `args.data.target_lang_id`?

Reply:
Good catch, I think it should be the latter one. Fixed